apps/concept-catalog/app/[catalogId]/search-page-client.tsx

```diff
@@ -331,6 +334,17 @@ export const SearchPageClient = ({
       <SearchHitsPageLayout
         buttonRow={
           <>
+            {changeRequestEnabled && (
+              <Button
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                as={Link}
+                variant='secondary'
+                href={`/${catalogId}/change-requests`}
+              >
+                {loc.changeRequest.changeRequest}
+              </Button>
```

Comment:
```suggestion
              <Link href={`/${catalogId}/change-requests`} passHref>
                <Button variant='secondary'>
                  {loc.changeRequest.changeRequest}
                </Button>
              </Link>  
```

Reply:
Ikke tillatt i følge HTML-spec'en:

_"The [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element can be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g., buttons or other links)."_
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element